services/FacilityService.php

```diff
@@ -220,38 +201,49 @@ public function update($data)
                 $data["facility_npi"],
                 $data["facility_code"],
                 $data["facility_taxonomy"],
-                $data["fid"]
+                $data["fid"],
+                $data["mail_street"],
+                $data["mail_street2"],
+                $data["mail_city"],
+                $data["mail_state"],
+                $data["mail_zip"],
+                $data['oid']
             )
         );
     }
 
     public function insert($data)
     {
-        $sql  = " INSERT INTO facility SET";
-        $sql .= "     name=?,";
-        $sql .= "     phone=?,";
-        $sql .= "     fax=?,";
-        $sql .= "     street=?,";
-        $sql .= "     city=?,";
-        $sql .= "     state=?,";
-        $sql .= "     postal_code=?,";
-        $sql .= "     country_code=?,";
-        $sql .= "     federal_ein=?,";
-        $sql .= "     website=?,";
-        $sql .= "     email=?,";
-        $sql .= "     color=?,";
-        $sql .= "     service_location=?,";
-        $sql .= "     billing_location=?,";
-        $sql .= "     accepts_assignment=?,";
-        $sql .= "     pos_code=?,";
-        $sql .= "     domain_identifier=?,";
-        $sql .= "     attn=?,";
-        $sql .= "     tax_id_type=?,";
-        $sql .= "     primary_business_entity=?,";
-        $sql .= "     facility_npi=?,";
-        $sql .= "     facility_code=?,";
-        $sql .= "     facility_taxonomy=?";
-
+        $sql  = " INSERT INTO facility SET
+             name=?,
+             phone=?,
+             fax=?,
+             street=?,
+             city=?,
+             state=?,
+             postal_code=?,
+             country_code=?,
+             federal_ein=?,
+             website=?,
+             email=?,
+             color=?,
+             service_location=?,
+             billing_location=?,
+             accepts_assignment=?,
+             pos_code=?,
+             domain_identifier=?,
+             attn=?,
+             tax_id_type=?,
+             primary_business_entity=?,
+             facility_npi=?,
+             facility_code=?,
+             facility_taxonomy=?,
+             mail_street=?,
+             mail_street2=?,
+             mail_city=?,
+             mail_state=?,
+             mail_zip=?,
+             oid=? ";
```

Comment:
@sjpadgett ,
btw, I like this way better than the one further above where you crunched the query into several lines (I realize you were told to do this); the way you did it directly above this post is really nice since much easier to code review and add query elements in the future.

Reply:
this is just a fyi, don't need to make any code changes

Reply:
sorry @sjpadgett , this was meant for @juggernautsei 
I need more coffee...

Reply:
@bradymiller agree to disagree, I suppose. I find the multi-line, 1-field-per-line method to be much harder to read, tougher on code reviews, and generally a less pleasant coding experience. Consistency is part of our code standards, we should decide on a way to do it and then stick to it.

Reply:
In this case, was very difficult to know that the one of the column names was duplicated in the compacted style. I only picked it up because I checked each column which was much harder to do when they were compacted than when they are kept in the vertical positioning.